REDCapInstrumentAndVariableSQLRenamer.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 namespace VUMC\REDCapInstrumentAndVariableSQLRenamer;
+include_once(__DIR__ . "/classes/MessagedHandler.php");
```

Comment:
It looks like you have a type in this class name / filename. It should be "MessageHandler", correct?

Reply:
Deleted this. It is not necessary but good catch!